tests/interface/TWCardanoTests.cpp

```diff
@@ -33,8 +36,51 @@ TEST(TWCardano, Address) {
     ASSERT_TRUE(TWCardanoAddressEqual(address.get(), address2.get()));
 }
 
-/*
-TEST(TWCardano, Sign) {
-    // Add signer tests here
+TEST(TWCardano, PlanAndSign) {
+    Proto::SigningInput input;
+    input.set_amount((uint64_t)(1.0 * 1000000));
+    input.set_to_address("Ae2tdPwUPEZ4V8WWZsGRnaszaeFnTs3NKvFP2xRse56EPMDabmJAJgrWibp");
+    input.set_change_address("Ae2tdPwUPEZLKW7531GsfhQC1bNSTKTZr4NcAymSgkaDJHZAwoBk75ATZyW");
+    auto utxo = input.add_utxo();
+    Data txid = parse_hex("59991b7aa2d09961f979afddcd9571ff1c637a1bc0dab09a7233f078d17dac14");
+    utxo->mutable_out_point()->set_txid(txid.data(), txid.size());
+    utxo->mutable_out_point()->set_index(6);
+    utxo->set_amount((uint64_t)(15.0 * 1000000));
```

Comment:
can we do test for max send amount?

Reply:
There are 2 max_amount tests in the C++ level tests, but I can create one here too.